### ISSUE TITLE
[Feature Request] Compatible with Vue 3?

### ISSUE THREAD

**Issue by frederikhors** [vuetifyjs/vuetify#12251]
### Problem to solve
_I apologize if it is written somewhere and I have not read it, I have searched._

Can I use this project with Vue 3?
Is there a temporary branch of work in progress that I can try?

### Proposed solution
I need to start a new project and need to find out more before starting.

Thank you.

**Comment by KaelWD** [comment]
Duplicate of #11162
